omod/src/main/java/org/openmrs/module/adminui/page/controller/myaccount/ChangeDefaultsPageController.java

```diff
@@ -68,6 +79,35 @@ public UserDefaults getUserDefaults(@RequestParam(value = "defaultLocale", requi
     	return new UserDefaults(defaultLocale, proficientLocales);
     }
 
+    public static Locale normalizeLocale(String localeString) {
```

Comment:
Let me add a comment on the method. The method is required to convert locale string to actual Locale object.